src/app-header.c

```diff
@@ -134,11 +134,15 @@ void app_header_init(AppHeader* ah)
 	g_signal_connect_swapped(prev, "clicked", (GCallback) &app_header_nav_prev, ah);
 	g_signal_connect_swapped(ah->btn_current, "clicked", (GCallback) &app_header_nav_current, ah);
 	g_signal_connect_swapped(ah->btn_next, "clicked", (GCallback) &app_header_nav_next, ah);
+	gtk_widget_set_tooltip_text(prev, "Previous");
+	gtk_widget_set_tooltip_text(ah->btn_current, "Current");
+	gtk_widget_set_tooltip_text(ah->btn_next, "Next");
 
 	ah->btn_menu = gtk_button_new();
 	gtk_button_set_image(GTK_BUTTON(ah->btn_menu), gtk_image_new_from_icon_name("open-menu-symbolic", GTK_ICON_SIZE_MENU));
 	g_signal_connect_swapped(ah->btn_menu, "clicked", (GCallback) &app_header_menu, ah);
 	gtk_header_bar_pack_end(GTK_HEADER_BAR(ah), ah->btn_menu);
+  gtk_widget_set_tooltip_text(ah->btn_menu, "Menu");
```

Comment:
I think at least this one is obvious, no? I haven't noticed this in any other programs.

Also, check indentation (clang-format should help)